.github/ISSUE_TEMPLATE/doc-request.yml

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Document
+description: Suggest document changes
+title: "[Doc] "
+labels: ["doc-required"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you very much for submitting suggestions to Pulsar. Please answer the following questions before submitting a doc issue.
+  - type: checkboxes
+    attributes:
+      label: Search before asking
+      description: >
+        Please make sure to search in the [issues](https://github.com/apache/pulsar/issues) first to see whether the same issue was reported already.
+      options:
+        - label: >
+            I had searched in the [issues](https://github.com/apache/pulsar/issues) and found no similar issues.
+          required: true
+  - type: textarea
```

Comment:
How about adding a question to locate the scope quickly?

Something like:

```
- type: dropdown
    attributes:
      label: Apache Pulsar Component
      description: |
        Which component do you find issues with?
      multiple: false
      options:
        - "Get started"
        - "Concept and architecture"
        - "Schema"
        - "Function"
        - "Connector"
        - "SQL"
        - "Tiered storage"
    validations:
      required: true
```

For `options`, I just name a few examples. I can provide a complete list if it makes sense to you.

Reply:
This comment applies to `bug` and `enhancement` templates as well.

Reply:
I'd prefer to postpone such change to a follow-up patch:

1. This is a new feature in addition to we change the form only.
2. It can make sense to doc issues, but a bug/enhancement can be related to multiple components, thus asking a contributor to do the sorting task is a bit overwhelmed. A committer can help with labeling an issue later, though.